### PR TITLE
Make the sleeping slowdown configurable.

### PR DESCRIPTION
**Pull request by Ionic**
As part of the first enhancement in https://github.com/ArcticaProject/nx-libs/issues/132, we want to make optional sleep delays adjustable.

This PR handles this.

Sleep delays are given in milliseconds from 0 (disable slowdowns completely) to UINT_MAX (although high values will probably not be useful.)

If necessary, we can also change this to take values in microseconds.

### CODE REVIEW

File: nx-X11/programs/Xserver/hw/nxagent/Display.c [hunk 1]
```diff
@@ -2169,6 +2169,7 @@ static int nxagentCheckForDefaultDepthCompatibility()
 {
   int dDepth;
 
+  return 1;
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/hw/nxagent/Display.c]: This and the below 'return's addressing deactivations of color depth checks should not be part of this commit, IMHO.

I wonder if Mario's original diff (the first commit of this PR) should appear in the nx-libs history, at all.

Comment (sunweaver) [inline nx-X11/programs/Xserver/hw/nxagent/Display.c]: not be part of this PR..., I mean...

**Comment by Ionic** [comment]
Rebased and force-pushed.
